chronicles-rulebook.md

```diff
@@ -2656,9 +2646,9 @@ Item production is the process by which characters are able to craft consumable
 ##### Fine Armor, Hardened Leather
    >This creates a finely made suit of armor that has a Armor rating of 30, with 5 Durability.
 ##### Shield Spikes
-   >This creates a set of Shield Spikes that can be applied to a Shield that will allow the character to Reflect Physical. Shield spikes have 3 charges. Only one Shield Spikes or Mirror Coating can be on a shield at a time. The call for this is “Reflect”
+   >This creates a set of Shield Spikes that can be applied to a Shield that will allow the character to Reflect Physical. Shield spikes have 1 charge. Only one Shield Spikes or Mirror Coating can be on a shield at a time. The call for this is “Reflect”
 ##### Mirror Coating
-   >This creates a Mirror Coating that can be applied to a Shield that will allow the character to Reflect Spell. Mirror Coatings have 3 charges. Only one Shield Spikes or Mirror Coating can be on a shield at a time. The call for this is “Reflect”
+   >This creates a Mirror Coating that can be applied to a Shield that will allow the character to Reflect Spell. Mirror Coatings have 1 charge. Only one Shield Spikes or Mirror Coating can be on a shield at a time. The call for this is “Reflect”
```

Comment:
Do we have a call on if Shield Spikes and Mirror Coating need to be prepared and if they take up a defense slot?